rocky/tests/test_landing_page.py

```diff
@@ -7,3 +13,18 @@ def test_landing_page_redirect(rf, client_member):
 
     response = LandingPageView.as_view()(request)
     assert response.status_code == 302  # Redirects to crisis-room
+
+
+LANG_LIST = [lang[0] for lang in settings.LANGUAGES]
+
+
+@pytest.mark.parametrize("language", LANG_LIST)
+def test_language_lang_attribute(rf, client_member, language):
+    activate(language)
+
+    response = AccountView.as_view()(
+        setup_request(rf.get("account_detail"), client_member.user), organization_code=client_member.organization.code
+    )
+    assert response.status_code == 200
+    assertContains(response, '<html lang="' + language + '"')
+    deactivate()  # must decativate otherwise further tests fail to fetch context in English
```

Comment:
Should probably be a generic setup and teardown for language features. See also pytest scope [fixtures](https://docs.pytest.org/en/6.2.x/fixture.html)